runtime/crab/src/pallets/evm.rs

```diff
@@ -176,24 +176,21 @@ impl DispatchValidateT<AccountId, RuntimeCall> for DarwiniaDispatchValidator {
 		origin: &AccountId,
 		call: &RuntimeCall,
 	) -> Option<fp_evm::PrecompileFailure> {
-		if let Some(root) = pallet_sudo::Pallet::<Runtime>::key() {
-			if origin == &root {
-				// Without validation for root role
-				return None;
-			} else if matches!(
-				call,
-				RuntimeCall::Assets(..)
-					| RuntimeCall::Vesting(..)
-					| RuntimeCall::Ethereum(..)
-					| RuntimeCall::EVM(..)
-					| RuntimeCall::MessageTransact(..)
-			) {
-				return Some(fp_evm::PrecompileFailure::Error {
-					exit_status: ExitError::Other("These pallet's calls are forbidden".into()),
-				});
-			}
+		if matches!(
+			call,
+			RuntimeCall::Assets(..)
+				| RuntimeCall::Vesting(..)
+				| RuntimeCall::Ethereum(..)
+				| RuntimeCall::EVM(..)
+				| RuntimeCall::MessageTransact(..)
+		) {
+			Some(fp_evm::PrecompileFailure::Error {
+				exit_status: ExitError::Other("These pallet's calls are forbidden".into()),
+			})
+		} else {
+			<() as DispatchValidateT<AccountId, RuntimeCall>>::validate_before_dispatch(
+				origin, call,
```

Comment:
What's the effect of this? Seems to be different with previous version?

Reply:
We used to use gnosis multisig to dispatch sudo call from EVM. Now, we don't.

Reply:
My bad, there is no difference.